intermediate-exercises/src/components/03-FetchGithubData.js

```
@@ -43,21 +33,40 @@ class UsernameForm extends Component {
       username: '',
       repos: [],
     };
+
+    this.fetchRepos = this.fetchRepos.bind(this);
+    this.handleOnChange = this.handleOnChange.bind(this);
+  }
+
+
+  fetchRepos() {
+    const { username } = this.state;
+    const url = `https://api.github.com/users/${username}/repos`
+    axios.get(url)
+      .then(({data}) => this.setState({ repos: data }));
```

Exceptions are not handled here.
Add catch block